app/src/main/java/com/example/chooseyouowncocktail_g2/ui/search/SearchFragment.kt

```diff
@@ -0,0 +1,71 @@
+package com.example.chooseyouowncocktail_g2.ui.search
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ArrayAdapter
+import android.widget.SearchView
+import android.widget.Toast
+import androidx.fragment.app.Fragment
+import com.example.chooseyouowncocktail_g2.databinding.FragmentSearchBinding
+
+
+class SearchFragment : Fragment() {
+
+    private var _binding: FragmentSearchBinding? = null
+
+    // This property is only valid between onCreateView and
+    // onDestroyView.
+    private val binding get() = _binding!!
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        /* val SearchViewModel =
+             ViewModelProvider(this).get(SearchViewModel::class.java)*/
+
+        _binding = FragmentSearchBinding.inflate(inflater, container, false)
+        val root: View = binding.root
+
+
+        return root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        val theCocktailList = arrayListOf<String>("Old Fashioned","Negroni","Daiquiri","Dry Martini","Margarita","Espresso", "MartiniWhiskey Sour"," Manhattan")
+        val adapter = activity?.let {
+            ArrayAdapter<String>(it, android.R.layout.simple_list_item_1, theCocktailList)
```

Comment:
L'adapter da usare qui non è questo ma lo stesso che viene usato in home, cioè quello che contiene le card